service/collector/cloudformation.go

```diff
@@ -0,0 +1,173 @@
+package collector
+
+import (
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+	clientaws "github.com/giantswarm/aws-operator/client/aws"
```

Comment:
Please see https://github.com/giantswarm/fmt/blob/master/go/imports.md and get the blocks straight. I discussed this with Pawel because it is a common issue in our reviews and we want to get this into the CI pipeline. There is an issue in the SIG RelEng board so we can work on this during SWH. See also https://github.com/giantswarm/giantswarm/issues/7485.

Reply:
got it!